public/Highways.php

```diff
@@ -28,6 +28,7 @@ function index() {
    * @param int $id The database ID of the highway you'd like to view.
    * @return [Highway] The highway requested.
    */
+  //Do NOT change the return statement to @return Highway [Highway] unless you want a bad day
```

Comment:
There's a story here... :-D
But the bigger question is: Why would you want to, that doesn't even make sense? ;-)